projects/fsastorefrontlib/src/cms-components/comparison-table/comparison-table-container/comparison-table-container.component.html

```diff
@@ -1,27 +1,30 @@
 <div class="products-wrapper d-md-flex justify-content-around mb-2">
   <ng-container *ngIf="component$ | async as component">
-    <ul
-      ngbNav
-      #nav="ngbNav"
-      [ngClass]="{ 'hide-tab-heading': tabs$.length === 1 }"
-      class="nav-tabs"
-      [(activeId)]="active"
-    >
-      <ng-container *ngFor="let tabElement of tabs$; let i = index">
-        <li ngbNavItem [ngbNavItem]="i" *ngIf="tabElement | async as tab">
-          <a ngbNavLink> {{ tab.title }}</a>
-          <ng-template ngbNavContent>
-            <ng-template
-              [cxComponentWrapper]="{
-                flexType: 'CMSComparisonTabComponent',
-                uid: tab.uid
-              }"
-            >
-            </ng-template>
-          </ng-template>
-        </li>
+    <ng-container *ngIf="initialTabs.length > 0">
+      <ng-container *ngIf="tabs$ | async">
```

Comment:
Did you try to combine this two conditions into one, something like this `*ngIf="initialTabs.length > 0 && (tabs$ | async)"` and remove one `ng-container` element?